e2e_playwright/shared/data_mocks.py

```diff
@@ -27,82 +27,79 @@
 random.seed(0)
 
 
-class TestCaseMetadata(NamedTuple):
+class CaseMetadata(NamedTuple):
```

Comment:
Just curious why we need to rename this `TestCaseMetadata` to `CaseMetadata`?

Reply:
The `Test` prefix has a special meaning when run with pytest. Pytest would trie to execute this as a test class, which it isn't. You can prevent that via the `__test__ = False` marker, but that created other typing issues. So it is easier to just rename this class so that it's not interpreted a  test class.